required/tools/array.dtx

```diff
@@ -1401,13 +1420,13 @@ Bug reports can be opened (category \texttt{#1}) at\\%
 %    =\@sharp= which is temporarily occupied with
 %    =\relax= during the build-up of the preamble.
 %    To remove unwanted spaces before and after the column text, we set
-%    an =\ignorespaces=  in front and a =\unskip= afterwards.
+%    an =\ignorespaces=  in front (see above) and a =\unskip= afterwards.
```

Comment:
... but only if the cell isn't a math cell.

Reply:
I've not changed anything here other than the code is now above the comment not below - do I need to change more?

Reply:
I know, I just meant it makes sense to say why \textonly@unskip ... we (or I) neglected that when it got introduced